frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-header/feature-flag-details-page-header.component.html

```diff
@@ -0,0 +1,6 @@
+<app-common-details-page-header-container
+    rootName = "feature-flags.title.text"
+    detailsName = {{ flagName }}
+    rootLink= "feature-flags.link.text"
+> </app-common-details-page-header-container>
+<app-common-details-page-header-container></app-common-details-page-header-container>
```

Comment:
line 6 looks like it needs to be removed

Reply:
resolved